### DISCUSSION TITLE
Cant type my account in the login bar

### DISCUSSION THREAD

**Discussion by Vortey** [ppy/osu#15575] [Q&A] [answered]
Every time I type in the login bar any of letters  wont pop up

**Comment by bdach** [comment]
are you on android, or on macOS using a japanese keyboard layout (like romaji or similar)?

**Reply by Vortey** [reply]
Yes Im on android and Im not using romaji keyboard Im using Microsoft keyboard

**Reply by peppy** [reply]
Switch to the system default keyboard.

**Reply by Vortey** [reply]
> Switch to the system default keyboard.

Thanks it worked

**Reply by NevoeiroX** [reply]
I can't type too and im on android using default keyboard

**Reply by Moltanicaa** [reply]
Yes, I was also informed about this issue.  The default keyboard doesn't work either

https://user-images.githubusercontent.com/59089206/141474531-c3eed9ae-42d6-43c8-b58d-5b9dd09ff034.mp4

**Reply by peppy** [reply]
This seems like a regression since some of the users reporting this previous had it working. Maybe to do with an SDL bump? Although are we even using anything SDL side for android?

**Reply by bdach** [reply]
No idea. I cannot reproduce here, although maybe I am just using the "good" keyboards.

**Reply by MATRIX-feather** [reply]
https://github.com/ppy/osu-framework/blob/a8429246cd22b686db420a89c049a57792f37923/osu.Framework.Android/Input/AndroidTextInput.cs#L42
Probably because `pending += text` is missing here?

**Reply by peppy** [reply]
Potentially, are you able to test that?

**Reply by anton2007** [reply]
> Potentially, are you able to test that?

I can try to compile game with it, but I don't know how I can do it on smartphone inside chroot

**Comment by gadasiu** [comment]
Same issue here, happened after updating to the latest version, was fine previously. 
Tried 4 different keyboards (Samsung, Fleksy, Google, Swiftkey), and I'm unable to paste the login/password into the window either.

**Comment by frenzibyte** [comment, marked as answer]
Now tracked in https://github.com/ppy/osu/issues/8264.
